### ISSUE TITLE
Responsibilities in test OP-display-popup

### ISSUE THREAD

**Issue by marsangr** [openid-certification/oidctest#45]
The note in the test says "You should get a popup user agent login window".

From my point of view it is the RP (in this case, the test tool) who should be responsible for creating the popup and me, the OP, to honor the display parameter just to optimize for popups.

**Comment by zandbelt** [comment]
This is more of a spec question than a certification one; @selfissued has agreed to comment.

**Comment by gffletch** [comment]
I agree. The RP is responsible for opening the popup window. The OP is responsible if the display=popup parameter is specified to ensure the UI works correctly within the popup window (e.g. provide scrollbars if the UI is larger than the size of the window).

**Comment by travisspencer** [comment]
Even if this is a test issue and not a protocol issue, it would be great to get some clarification on `display=popup` in the upcoming errata 2, @selfissued. The wording in the spec makes it not entirely clear who's responsible for what WRT the popup.

**Comment by gb056** [comment]
What's the status on this? I can't see how the OP is capable of opening a dialog window, let alone opening one that is an appropriate size for the RP (of which it knows nothing about). It sounds to me like this test should be opening the popup itself and that the OP should be rendering a login view that scales appropriately.
